admin_guide/disabling_features.adoc

```diff
@@ -0,0 +1,276 @@
+
+| *CRIContainerLogRotation*
+| Enables container log rotation for the CRI container runtime.
+
+| *CSIPersistentVolume*
+| Enables discovering and mounting volumes provisioned through a CSI (Container Storage Interface) compatible volume plugin. For more information,
+see the link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md[CSI Volume Plugins in Kubernetes Design Doc].
+
+| *CustomPodDNS*
+| Enables customizing the DNS settings for a pod using the *dnsConfig* property. For more information, see
+xref:../install_config/master_node_configuration.adoc#master-config-serving-information-config[Serving Information Configuration].
+
+| *CustomResourceSubresources*
+| Enables */status* and */scale* subresources on resources created from CustomResourceDefinition. 
+See xref:../admin_guide/custom_resource_definitions.adoc#admin-guide-custom-resources[Extending the Kubernetes API with Custom Resources].
+
+| *CustomResourceValidation*
+| Enables schema-based validation on resources created from a custom resource definition.
+For more information, see xref:../admin_guide/custom_resource_definitions.adoc#admin-guide-custom-resources[Extending the Kubernetes API with Custom Resources].
+
+| *DebugContainers*
+| Enables running a debugging container in a pod namespace to troubleshoot a running Pod.
+
+| *DevicePlugins*
+| Enables xref:../dev_guide/device_plugins.adoc#using-device-plugins[device plug-in-based] resource provisioning on nodes.
+
+| *DynamicKubeletConfig*
+| Enables the dynamic configuration in a cluster.
+
+| *DynamicVolumeProvisioning(deprecated)*
+| Enables the dynamic provisioning of persistent volumes to pods.
+
+| *EnableEquivalenceClassCache*
+| Enables the scheduler to cache equivalence of nodes when scheduling Pods.
+
+| *ExperimentalCriticalPodAnnotation*
+| Enables annotating specific pods as critical so that their scheduling is guaranteed.
+
+| *ExperimentalHostUserNamespaceDefaultingGate*
+| Enables the disabling of user namespaces. This is for containers that are using other host projects, host mounts, or containers that are privileged or using specific non-project capabilities, such as MKNODE, SYS_MODULE, and so forth. This should only be enabled if user project remapping is enabled in the Docker daemon.
+
+| *GCERegionalPersistentDisk*
+| Enables the xref:../install_config/persistent_storage/persistent_storage_gce.adoc#install-config-persistent-storage-persistent-storage-gce[GCE
+Persistent Disk] feature.
+
+| *HugePages*
+| Enables the allocation and consumption of pre-allocated xref:../scaling_performance/managing_hugepages.adoc#scaling-performance-managing-huge-pages[huge pages].
+
+| *HyperVContainer*
+| Enables Hyper-V isolation for Windows containers.
+
+| *Intializers*
+|Enables the xref:../architecture/additional_concepts/dynamic_admission_controllers.adoc#architecture-additional-concepts-dynamic-admission-controllers[dynamic admission control] 
```

Comment:
In some entries, there is a space between the text and the |, but not here. I would check consistency across all.